includes/Form.class.php

```diff
@@ -165,6 +165,18 @@ function process($params) {
 		}
 		$email = $_POST["email"];
 		$phone = isset($_POST["phone"]) ? $_POST["phone"] : "";
+		$lists = (isset($_POST["nlbox"]) && $_POST["nlbox"]) ? $_POST["nlbox"] : array();
+		if (isset($_SESSION["image_random_value"])) {

```

Comment:
This is not actually being sent to the AC servers - we really only validate it within the PHP wrapper, which is installed on the external site. It seems to work as expected during tests.